### DISCUSSION TITLE
SQL variables - scope (macro) ?

### DISCUSSION THREAD

**Discussion by jimdickson007** [duckdb/duckdb#14011] [Q&A] [answered]
I want to define macro1 which calls macro2
macro2 needs to accept argument - which I want to be an SQL variable

Is this possible ?

**Comment by Tishj** [comment]
Have you tried and are reporting an issue you encountered there?

**Reply by jimdickson007** [reply]
yes, tried and failed.
If you need a test case, will try to create one over next few days.

**Comment by jimdickson007** [comment, marked as answer]
Ignore me - cannot reproduce, must have a different issue (brain fog).
